tgui/packages/tgui/interfaces/MusicInstrument.tsx

```diff
@@ -0,0 +1,183 @@
+
+          if (ev.key === 'Control') {
+            toggleKeybind();
+          }
+        }}>
+        <Box className="instrument__keyboardwrapper">
+          <Box className="instrument__outerpanel">
+            <Box className="instrument__speaker" />
+            <Flex direction="column">
+              <Box className="instrument__panel">
+                <Box className="instrument__keyboardsupport">
+                  <Button
+                    className="instrument__toggle-keyboard-button"
+                    title="Toggle keyboard support (toggle with ctrl)"
+                    onClick={() => toggleKeybind()}
+                    icon="keyboard"
+                  />
+                  <Box
+                    className="instrument__keybind-indicator"
+                    style={{
+                      'box-shadow': `0px 0px 5px ${keybindToggle ? '#1b9b37' : '#5a1919'}`,
+                      'background': `${keybindToggle ? '#1b9b37' : '#5a1919'}`,
+                    }}
+                  />
+                </Box>
+                <Box className="instrument__panel-input">
+                  <Knob
+                    animated
+                    stepPixelSize={24}
+                    minValue={-24}
+                    maxValue={24}
+                    value={keyOffset}
+                    onDrag={(e, v) => setKeyOffset(v)}
+                    title={'Keybind offset'}
+                  />
+                  <span>Offset</span>
+                </Box>
+                <Box className="instrument_panel-info">
+                  <h1 style={{ 'text-align': 'center' }}>{name.toUpperCase()}</h1>
+                </Box>
+                <Box className="instrument__panel-input">
+                  <Knob
+                    animated
+                    stepPixelSize={1}
+                    minValue={0}
+                    maxValue={100}
+                    title={'Volume'}
+                    value={volume}
+                    onDrag={(e, v) => setVolume(v)}
+                  />
+                  <span>Volume</span>
+                </Box>
+                <Box className="instrument__panel-input">
+                  <Knob
+                    animated
+                    stepPixelSize={6}
+                    minValue={-12}
+                    maxValue={12}
+                    title={'Transpose'}
+                    value={transpose}
+                    onDrag={(e, v) => setTranspose(v)}
+                  />
+                  <span>Transpose</span>
+                </Box>
+              </Box>
+              <Box className="instrument__keyorder">
+                <h4>Key binding order for keyboard input</h4>
+                <Input
+                  className="instrument__input_keyorder"
+                  value={noteKeysOrder.join('')}
+                  onInput={(e, v) => setNoteKeysOrder(v.split(''))}
+                />
+                <h6>Type in the order you wish the keybindings to be placed</h6>
```

Comment:
Don't use `h#` tags for non-headers.

Reply:
Alright, I'll change it

Reply:
87adf213466494695784c587d11155051c7de011